### ISSUE TITLE
Destructored vars (const, let, var) do not need type

### ISSUE THREAD

**Issue by shall-git** [gajus/eslint-plugin-flowtype#375]
```javascript
const {links, history} = this.props
```
**Expected**: No error.

**Actual**: ESlint: missing variable type  annotation. (flowtype-/require-variable-type)

Maybe I'm missing something, but I can't figure out a valid configuration for this case

    "flowtype/require-variable-type": [
      2,
      {
        "excludeVariableMatch": "^map", // mapStateToProps & mapDispatchToProps
        "excludeVariableTypes": {
          "let": false,
          "const": false
        }
      }
    ]
